template/styles/docfx.js

```diff
@@ -54,17 +57,35 @@ $(function () {
     $('.article-container a:not([class*="no-external-icon"])[href^="http"]')
       .each(function (i, anchor) {
         const anchorChildren = $(anchor).children();
-        if(anchorChildren.length > 0) {
+        if (anchorChildren.length > 0) {
           const lastChild = anchorChildren[anchorChildren.length - 1];
           $(lastChild).addClass('external-link');
           $(anchor).addClass('external-link-parent');
         } else {
           $(anchor).addClass('external-link');
         }
-    })
+      })
+  }
+
+  function addGtmButtons() {
+
+    if (!$(".sample-container:first + p>a.trackCTA").length) {

```

Comment:
```suggestion
    if ($(".sample-container").length && !$(".sample-container:first + p>a.trackCTA").length) {

```
We should check also if there are any sample-containers in the topic